ecosystem/lava-sdk/src/stateTracker/state_tracker.ts

```diff
@@ -89,17 +94,18 @@ export class StateTracker {
     // Run all updaters
     // Only for chain query
     if (this.stateQuery instanceof StateChainQuery) {
-      await this.update();
+      await this.update(0);
     }
 
     // Fetch Pairing
-    this.timeTillNextEpoch = await this.stateQuery.fetchPairing();
+    [this.timeTillNextEpoch, this.virtualEpoch] =
```

Comment:
let's have only one source of truth for the virtual epoch, not save it in several places, should be emergency mode tracker